### DISCUSSION TITLE
Feasibility of creating web directory based on dialy.dev and contributing back

### DISCUSSION THREAD

**Discussion by ifoi** [dailydotdev/daily#732] [General]
First of I love daily.dev.   

I am about to start work on a project, that is similar to daily.dev, except that it will be directory of tech job resources  to help folks looking to get into tech,and would be adding a few features, that we would then like to contribute back to daily.dev .

We are new to open source contribution, and are wondering what would be the best way to do this?    or any suggestion for us to get started right.

We are wondering what folks thoughts are on the feasibility of creating a wed directory based on daily.dev   as opposed to starting from scratch.   Since daily.dev already has many of the features we would like to implement.   

here is one of our user stories:  
 
#### Implement voting/rating/up-voting
 - Acceptance criteria - use able to upvote resources and comments. 

#### Commenting feature   
-   Acceptance criteria - users able to add comments on resources, and reply to others comments (preferably threaded ) - have option to report or flag comment for Admin review.  

#### Social sharing feature   
- Acceptance criteria - clicking on share button pops up screen to allow sharing to social media, or just to copy link.
Tag feature   
 Acceptance criteria -   logged in  User (or Admin)  is able to add tags to each resource  - clicking on tag will lead them to more related resources. 

####  Site wide search  
-  Acceptance criteria -  user able to search by keywords, or tags. 

Thanks in advance,

Ifo

**Comment by ifoi** [comment]
My apologies for accidentally creating a duplicate post.  I only meant to edit the original.     and apparently i can't delete a duplicate comment on github https://github.com/community/community/discussions/3608

**Comment by idoshamun** [comment]
Hi @ifoi,

I'm not sure I understand your question. Would you like to add your proposal as a feature to daily.dev that will help folks find their next job?

**Reply by ifoi** [reply]
I was looking to create those  features, for a project that i am working on.   and then open source the code, so that it could also be used on daily.dev, If they do decide to host such a resource.
   
   Or just have it as an example of something that can be built from the daily.dev codebase .
   
   Does that make sense?

**Reply by idoshamun** [reply]
Our codebase wasn't designed in a way that will allow changing the use case quickly. I believe that it will be easier to start from scratch and get inspiration from our codebase other than actually fork it for a new project. I hope that makes sense.

**Reply by ifoi** [reply]
Yes that makes sense.  Thanks..
